cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -181,4 +217,43 @@ class UploadResultToSw360Command : CliktCommand(
 
     private fun createReleaseName(pkgId: Identifier) =
         listOf(pkgId.namespace, pkgId.name).filter { it.isNotEmpty() }.joinToString("/")
+
+    private fun uploadAttachmentToRelease(
+        sw360ReleaseClient: SW360ReleaseClientAdapter,
+        release: SW360Release,
+        path: Path,
+        type: SW360AttachmentType
+    ): SW360Release? {
```

Comment:
Can this function actually return *null*?

Reply:
No 🤣